PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetEnergyFlow.cxx

```diff
@@ -109,7 +109,7 @@
   Bool_t AliAnalysisTaskEmcalJetEnergyFlow::Run()
   {
           if(fOutput->IsUseScaling()) Printf("Scaling is enabled! \n");
-  
+          if(IsMCprod) Printf("MC production");
```

Comment:
Since this can't be turned off (as in the case of AliDebugStream, for example) and it runs every event, this will inflate the logs on the LEGO trains. Please remove this line or use something like AliDebugStream

Reply:
I will move this to ExecOnce(), so this will avoid filling the log file and it makes sense since I don't expect the flag's value to change per event. Similarly I will also move the preceding line to ExecOnce(). Thank you for checking!

Reply:
Very good, thanks!

Reply:
AliAnalysisTaskEmcal provides UserExecOnce as framework function which user tasks can use to implement custom initialisations, please use that instead and never overwrite framework functions like ExecOnce directly.